assesspy/metrics.py

```diff
@@ -178,7 +180,7 @@ def prb(
         ap.prb(ap.ccao_sample().estimate, ap.ccao_sample().sale_price)
     """
     prb_model = _calculate_prb(estimate, sale_price)
-    prb = float(prb_model.params[0])
+    prb = float(prb_model.params[1])
```

Comment:
**[Suggestion, non-blocking]** Since the order of coefficients has proven a little bit tricky, perhaps we can persist a comment explaining our choice?

```suggestion
    # Get the coefficient from the OLS model.
    # We select element 1, since element 0 is the intercept 
    prb = float(prb_model.params[1])
```

Reply:
Done in 065dfcf!